api/v1alpha1/fio_types.go

```diff
@@ -22,23 +22,32 @@ import (
 
 // FioSpec defines the desired state of Fio
 type FioSpec struct {
```

Comment:
can we please add the Affinity related structs?

Reply:
sure, but can we please do it in a separate PR, this is getting big

Reply:
sure. can you add a TODO line for that? the bot will take care of issue creation then.